test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -71,3 +86,21 @@ def test_mfcc(self, log_mels):
         transform = T.MFCC(sample_rate=sample_rate, log_mels=log_mels)
         waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform])
+
+    @nested_params(
+        [0.7, 0.8, 0.9, 1.0, 1.3],
+        [True, False],
+    )
+    def test_timestretch(self, rate, test_complex):
+        transform = T.TimeStretch(fixed_rate=rate)
+        spectrogram = get_spectrogram(return_complex=test_complex)
+        self.assert_grad(transform, [spectrogram])
+
+    @nested_params(
+        [0.7, 0.8, 0.9, 1.0, 1.3],
```

Comment:
@anjali411

So the finding is that shrinking the spectrogram in time axis with factor of `0.7` causes 2nd order gradient to be `NaN`, while it works for the other parameters.
What do you think is a good way to communicate this to users?

Reply:
Did it return `NaN`s before we merged the PR migrating `TimeStretch` to complex types? If not, one or more of `abs`, `angle` or `polar` is likely causing the issue.

Reply:
TODO: @mthrok to file an issue in PyTorch core for the sake of record.

Reply:
> Did it return `NaN`s before we merged the PR migrating `TimeStretch` to complex types? 

Testing this in #1430

Reply:
@anjali411 

I ran the test with the code with pseudo complex code and found that the same tests fail for not only `0.7` but also for `0.8`, `0.9` and `1.3`. So This suggests that complex Tensor made these function support autograd.

https://app.circleci.com/pipelines/github/pytorch/audio/5655/workflows/4c1f6dc6-deb6-4730-815f-31946b9af73d/jobs/194026

Reply:
That's interesting. Could you please look into which function is breaking the test (return `nan`s instead of the non `nan` expected value) for the input the test is being ran on? It would be nice to file an issue both for real and complex operators which are returning incorrect value on edge cases!

Reply:
Filed the issue https://github.com/pytorch/pytorch/issues/55557